README.md

```diff
@@ -27,29 +28,30 @@ To sign requests made with pre-5 versions of the clients the following intercept
 
 ```java
 import java.io.IOException;
+
 import org.apache.http.HttpRequestInterceptor;
 import org.apache.http.client.ClientProtocolException;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
 import io.github.acm19.aws.interceptor.http.AwsRequestSigningApacheInterceptor;
 import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
-import software.amazon.awssdk.auth.signer.Aws4Signer;
+import software.amazon.awssdk.http.auth.aws.signer.AwsV4HttpSigner;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.utils.IoUtils;
 
 final class Example {
     public static void main(String[] args) throws ClientProtocolException, IOException {
         HttpRequestInterceptor interceptor = new AwsRequestSigningApacheInterceptor(
-                "service",
-                Aws4Signer.create(),
-                DefaultCredentialsProvider.create(),
-                Region.US_WEST_2
+            "service",
```

Comment:
8 spaces of indentation is more conventional for wrapping a line: https://www.oracle.com/technetwork/java/codeconventions-150003.pdf